core/runtime/src/main/java/io/quarkus/runtime/logging/LogCleanupFilter.java

```diff
@@ -21,13 +21,17 @@ public LogCleanupFilter(List<LogCleanupFilterElement> filterElements) {
 
     @Override
     public boolean isLoggable(LogRecord record) {
-        if (record.getLevel().intValue() != Level.INFO.intValue())
+        if (record.getLevel().intValue() != Level.INFO.intValue()
+                && record.getLevel().intValue() != Level.WARNING.intValue()) {
```

Comment:
I think maybe this should be `record.getLevel().intValue() < Level.INFO.intValue()`, if I understand the intent correctly (to return `true` in the event of `DEBUG`, `TRACE` etc.).  In any event, effective levels are integer values so it should be a range check rather than a check for two exact values in within the range.

Reply:
So this is not my code but I think the intent was to allow filtering only for `INFO` messages.

You're right that we should be more flexible but I think it should be:
```java
        if (record.getLevel().intValue() > Level.WARNING.intValue()) {
            return true;
        }
```

I added a commit to address your concern.